python/tvm/meta_schedule/builder/local_builder.py

```diff
@@ -149,8 +146,14 @@ def build(self, build_inputs: List[BuilderInput]) -> List[BuilderResult]:
         results: List[BuilderResult] = []
         map_result: MapResult
 
+        pool = PopenPoolExecutor(
```

Comment:
can we add some comment here just to show why we use `PopenPool` here differently than how we use it in RPCRunner and LocalRunner?